docs/scripts.md

```diff
@@ -0,0 +1,26 @@
+# Truffle Scripts
+
+The ```./scripts``` folder contains Truffle scripts (JS files) which can be executed in any Ethereum testnet or mainnet.
+
+## Prerequisites
+
+Before executing any script, verify if the Zero Collateral contracts are deployed in the network where you want to execute it.
+
+The addresses are located at ```./config/networks/NETWORK/zerocollateral.js``` where ```NETWORK``` is the network where you want to execute the scripts.
+
+> The network names are configured in the Truffle config file. Please, see details ```./truffle-config.js``` file (section ```networks```).
+> If the contracts are not defined in the network you are looking for, please contact any team member.
```

Comment:
`If the contracts are not defined in the network `
-->
`If the contracts have not been deployed on the network `